### ISSUE TITLE
About RL loss

### ISSUE THREAD

**Issue by Ricca-xie** [lyeskhalil/CORL#1]
reinforce_loss = ((cost.squeeze(1) - bl_val) * log_likelihood).mean()
loss = reinforce_loss + bl_loss - opts.ent_rate * e

Thanks for your code sharing, but I'm confused with the loss calculation that why using the entropy, and can you give me some explanation or suggestion so I can understand it. (the above codes are from train.py, and train_batch function)

**Comment by Ricca-xie** [comment]
I just got it, 😀
